dotCMS/src/integration-test/java/com/dotcms/scheduler/TestSynchronizedTask.java

```diff
@@ -0,0 +1,36 @@
+package com.dotcms.scheduler;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import io.vavr.control.Try;
+
+public class TestSynchronizedTask extends DotSyncronizedTask {
+
+    private final static String SYSTEM_INCREMENT_KEY = "TestSynchronizedTaskKey";
+    private final static String TASK_RUNNING_KEY = "TestSynchronizedTaskRunning";
+    public static int getTotalIncrement() {
+        return Try.of(() -> Integer.parseInt(System.getProperty(SYSTEM_INCREMENT_KEY))).getOrElse(0);
+    }
+    
+    final int myIncrement;
+    
+    public TestSynchronizedTask(int inc) {
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)